packages/web-app-webfinger/src/index.ts

```diff
@@ -0,0 +1,42 @@
+import translations from '../l10n/translations.json'
+import Resolve from './views/Resolve.vue'
+
+// just a dummy function to trick gettext tools
+function $gettext(msg) {
+  return msg
+}
+
+const appInfo = {
+  name: $gettext('Webfinger'),
+  id: 'webfinger',
+  icon: 'fingerprint',
+  isFileEditor: false
+}
+
+const routes = () => [
+  {
+    name: 'root',
+    path: '/',
+    redirect: () => {
+      return { name: 'webfinger-resolve' }
+    },
+    meta: {
```

Comment:
probably too early here 😁 I'm planning to allow defining a root route of an app, so that we don't need to rely on apps having a navItem (which is used right now to identify the first route of an app when setting the app as `options.defaultExtension`).